pkg/eventingtls/servermanager.go

```diff
@@ -61,23 +60,30 @@ func NewServerManager(ctx context.Context, httpReceiver, httpsReceiver *kncloude
 }
 
 // Blocking call. Starts the 2 servers
-func (s *ServerManager) StartServers(ctx context.Context) (httpError, httpsError error) {
+func (s *ServerManager) StartServers(ctx context.Context) error {
 	// start servers
-	wg := sync.WaitGroup{}
-	wg.Add(2)
+	errCh := make(chan error)
```

Comment:
I think we need buffering otherwise whatever go routing exits last will hang on sending the error

Reply:
In line 82, we "clear" the channel (and then return the error). With this approach we would get the first error only anyhow, or what am I missing?

Reply:
hmm, is that what happens?

https://go.dev/play/p/YyizRrOz16d

Reply:
hmmm. Ok. I agree :thinking:

Reply:
updated it